src/linker/Linker/TypeReferenceExtensions.cs

```diff
@@ -65,16 +74,16 @@ internal static void ParseGenericParameters (IList<GenericParameter> genericPara
 			sb.Insert (0, '<');
 		}
 
-		private static void ParseGenericArguments (Stack<TypeReference> genericArguments, int argumentsToTake, StringBuilder sb)
+		static void PrependGenericArguments (Stack<TypeReference> genericArguments, int argumentsToTake, StringBuilder sb)
 		{
-			sb.Insert (0, '>').Insert (0, genericArguments.Pop ().GetDisplayName ());
+			sb.Insert (0, '>').Insert (0, genericArguments.Pop ().GetDisplayNameWithoutNamespace ().ToString ());
```

Comment:
I thought we were dropping namespaces from types in places other than the "main member" being printed.